client/src/main/java/com/alibaba/nacos/client/logging/AbstractNacosLogging.java

```diff
@@ -31,13 +33,13 @@
     
     private static final String NACOS_LOGGING_DEFAULT_CONFIG_ENABLED_PROPERTY = "nacos.logging.default.config.enabled";
     
-    private static final String NACOS_LOGGING_PATH_PROPERTY = "nacos.logging.path";
+    private static final String NACOS_LOGGING_PATH_PROPERTY = "JM.LOG.PATH";
     
     static {
         String loggingPath = System.getProperty(NACOS_LOGGING_PATH_PROPERTY);
         if (StringUtils.isBlank(loggingPath)) {
             String userHome = System.getProperty("user.home");
-            System.setProperty(NACOS_LOGGING_PATH_PROPERTY, userHome + "/logs/nacos");
+            System.setProperty(NACOS_LOGGING_PATH_PROPERTY, userHome + File.separator + "logs");
```

Comment:
I think the `nacos` need keep, because in user.home/logs, there are many different app will print logs.

Reply:
I do kept the 'nacos' in nacos-log4j.xml, as is shown in the next comment.

The reason why I move 'nacos' dir into xml files is:
In 1.3.2, the ${nacos.logging.path} is directly the parent dir of log files: 
`if ${nacos.logging.path} exists: `
`    directly use ${nacos.log.path} as the log dir`
`else:`
`    use ${nacos.log.path} = ${user.home}/logs/nacos as the log dir`
[code 1]
but in 1.4.0, the ${JM.LOG.PATH} is NOT the direct parent dir of log files. The direct dir should be ${JM.LOG.PATH}/nacos
I can't direcly write this way:
`if ${JM.LOG.PATH} exists: `
`    ${JM.LOG.PATH} = ${JM.LOG.PATH}/nacos`
`else:`
`    use ${JM.LOG.PATH}= ${user.home}/logs/nacos as the log dir`
[code 2]
this code is wrong, because if ${JM.LOG.PATH} already exists, it should not be reassigned.
so I write this way:
`if ${JM.LOG.PATH} exists: `
`    do nothing`
`else:`
`    ${JM.LOG.PATH}= ${user.home}/logs`
Then in the resource files:
Use ${JM.LOG.PATH}/logs instead of ${JM.LOG.PATH} as the logging parent dir.
But I see this process is not elegant.